constrained_ik/CHANGELOG.rst

```diff
@@ -0,0 +1,11 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package constrained_ik
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.1.1 (2017-03-24)
+------------------
+
```

Comment:
This came from git-cherry-picking, I can update the change logs once I try to get it ready for release.